src/equations/compressible_euler_1d.jl

```diff
@@ -386,43 +388,43 @@ function signature with argument `which` set to `Val{:minus}()` or `Val{:plus}`.
 """
 @inline function splitting_steger_warming(u, orientation::Integer,
                                           equations::CompressibleEulerEquations1D)
-  fm = splitting_steger_warming(u, Val{:minus}(), orientation, equations)
-  fp = splitting_steger_warming(u, Val{:plus}(),  orientation, equations)
-  return fm, fp
+    fm = splitting_steger_warming(u, Val{:minus}(), orientation, equations)
+    fp = splitting_steger_warming(u, Val{:plus}(), orientation, equations)
+    return fm, fp
 end
 
 @inline function splitting_steger_warming(u, ::Val{:plus}, orientation::Integer,
                                           equations::CompressibleEulerEquations1D)
-  rho, rho_v1, rho_e = u
-  v1 = rho_v1 / rho
-  p = (equations.gamma - 1) * (rho_e - 0.5 * rho_v1 * v1)
-  a = sqrt(equations.gamma * p / rho)
+    rho, rho_v1, rho_e = u
+    v1 = rho_v1 / rho
+    p = (equations.gamma - 1) * (rho_e - 0.5 * rho_v1 * v1)
+    a = sqrt(equations.gamma * p / rho)
 
-  lambda1 = v1
-  lambda2 = v1 + a
-  lambda3 = v1 - a
+    lambda1 = v1
+    lambda2 = v1 + a
+    lambda3 = v1 - a
 
-  lambda1_p = positive_part(lambda1) # Same as (lambda_i + abs(lambda_i)) / 2, but faster :)
-  lambda2_p = positive_part(lambda2)
-  lambda3_p = positive_part(lambda3)
+    lambda1_p = positive_part(lambda1) # Same as (lambda_i + abs(lambda_i)) / 2, but faster :)
+    lambda2_p = positive_part(lambda2)
+    lambda3_p = positive_part(lambda3)
 
-  alpha_p = 2 * (equations.gamma - 1) * lambda1_p + lambda2_p + lambda3_p
+    alpha_p = 2 * (equations.gamma - 1) * lambda1_p + lambda2_p + lambda3_p
 
-  rho_2gamma = 0.5 * rho / equations.gamma
-  f1p = rho_2gamma * alpha_p
-  f2p = rho_2gamma * (alpha_p * v1 + a * (lambda2_p - lambda3_p))
-  f3p = rho_2gamma * (alpha_p * 0.5 * v1^2 + a * v1 * (lambda2_p - lambda3_p)
-                      + a^2 * (lambda2_p + lambda3_p) * equations.inv_gamma_minus_one)
+    rho_2gamma = 0.5 * rho / equations.gamma
+    f1p = rho_2gamma * alpha_p
+    f2p = rho_2gamma * (alpha_p * v1 + a * (lambda2_p - lambda3_p))
+    f3p = rho_2gamma * (alpha_p * 0.5 * v1^2 + a * v1 * (lambda2_p - lambda3_p)
+           + a^2 * (lambda2_p + lambda3_p) * equations.inv_gamma_minus_one)
```

Comment:
This is a bit surprising since it belongs to the open parenthesis in `(alpha_p`. But I guess this is okay...

Reply:
Yes, and it seems unfixable. Resolving this...